vueui/src/components/view/machinery/vending.vue

```diff
@@ -188,4 +188,8 @@ p.danger {
   opacity: 1;
 }
 
-</style>
\ No newline at end of file
+#content > div {
+  overflow-x: hidden;
```

Comment:
Any chance that this would "silently hide" content?

Reply:
Probably not. I specifically put it in the scoped SCSS of vending machines so that the scrollbar wouldn't appear for the vending UI alone, since the vending UI is `display:flex` and grows with the viewport.